docs/intro.rst

```diff
@@ -314,17 +314,17 @@ This is the current list of error and warning codes:
 +------------+----------------------------------------------------------------------+
 | **E3**     | *Blank line*                                                         |
 +------------+----------------------------------------------------------------------+
-| E301       | expected 1 blank line, found 0                                       |
+| E301       | expected 1 blank line, found …                                       |
```

Comment:
So, what do you want? just removal of ", found 0"? Where? in E301 or in all E*? Or replacing with "..." (ascii)?